README.md

```diff
@@ -35,3 +35,9 @@ The following environment variables can be passed in to customize run-time funct
 - HTTP_PORT - port used by http server (default 80)
 - DHCP_RANGE - dhcp range to use for provisioning (default 172.22.0.10-172.22.0.100)
 - MARIADB_PASSWORD - The database password
+
+The following environment variables can be used to override the settings in the ironic configuration file:
```

Comment:
I would make it less specific I guess, saying that configuration can be overriden by environment variables. Then say as an example the following (and show the config options so that it is understandable how to extend this to other configuration options)

Reply:
Fixed, thank you for feedback